lib/handler/mruby.c

```diff
@@ -62,6 +62,12 @@ static void setup_globals(mrb_state *mrb)
 
     h2o_mruby_eval_expr(mrb, "$LOAD_PATH << \"#{$H2O_ROOT}/share/h2o/mruby\"");
     h2o_mruby_assert(mrb);
+
+    /* require and include built-in libraries */
+    h2o_mruby_eval_expr(mrb, "require \"h2o.rb\"\n"
+                             "require \"acl.rb\"\n"
+                             "include H2O::ACL\n");
```

Comment:
Just one more thing...

How about doing all of the following?
1. renaming h2o.rb to bootstrap.rb
2. require and include the acl code _from_ bootstrap.rb